packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.controller.js

```diff
@@ -0,0 +1,111 @@
+import flatten from 'lodash/flatten';
+import map from 'lodash/map';
+import filter from 'lodash/filter';
+import illustration from '../assets/partner.png';
+
+export default class PciTrainingDashboardController {
+  /* @ngInject */
+  constructor(CucCloudMessage, CucRegionService, atInternet) {
+    this.CucCloudMessage = CucCloudMessage;
+    this.CucRegionService = CucRegionService;
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.illustration = illustration;
+    // eslint-disable-next-line prefer-destructuring
+    this.currentRegion = this.regions[0];
+
+    this.loadMessages();
+    this.resourceUsage = flatten(
+      map(
+        filter(this.usage.resourcesUsage, {
+          type: 'ai-serving-engine',
+        }),
+        'totalPrice',
+      ),
+    ).reduce((a, b) => a + b, 0);
+
+    this.eaiDocsUrl = '';
+    this.runningJobs = this.getJobsWithSelector((job) => job.isRunning());
```

Comment:
This looks like an override of filter function, I'm not sure it is necessary

Reply:
we tried to avoid the same code.